src/engine/renderer/tr_cmds.cpp

```diff
@@ -252,6 +250,12 @@ R_AddSetupLightsCmd
 */
 void R_AddSetupLightsCmd()
 {
+	if ( !glConfig2.realtimeLighting ||
+	     r_realtimeLightingRenderer.Get() != Util::ordinal( realtimeLightingRenderer_t::TILED ) )
+	{
+		return;
+	}
```

Comment:
Totally nitpicking but I prefer that syntax:

```c++
	if ( !glConfig2.realtimeLighting )
	{
		return;
	}
	
	if ( r_realtimeLightingRenderer.Get() != Util::ordinal( realtimeLightingRenderer_t::TILED ) )
	{
		return;
	}
```

This is the pattern we usually implement (and I usually write new code or modify code this way), for example:

https://github.com/DaemonEngine/Daemon/blob/926d8f6680af598efe49eea791a27567da3ad7f9/src/engine/renderer/tr_scene.cpp#L288-L301

Reply:
I see it as one condition. "Is the tiled renderer active"